### PR TITLE
adding file with things to do in Milwaukee Jianchengit milwaukee

### PR DESCRIPTION
**Pull request by jianchengit**

### CODE REVIEW

**Review by jianchengit — COMMENTED** [review]
looks good.

**Comment by szimmer116** [comment]
@jianchengit this is a review

File: jianchengit-milwaukee.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Milwaukee
+
+### Places to eat
+
+### Things to do
```
Comment (RaveeshK) [inline jianchengit-milwaukee.md:7]: Looks great!!

File: jianchengit-milwaukee.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Milwaukee
+
+### Places to eat
```
Comment (nkmadhav1) [inline jianchengit-milwaukee.md:3]: How do i get to milwaukee from RW :-)

**Review by RaveeshK — COMMENTED** [review]
Looks great!

**Comment by RaveeshK** [comment]
@jianchengit  Looks great!

**Pull request merged**